src/Yesod/Auth/OIDC.hs

```diff
@@ -335,9 +336,10 @@ findProvider loginHint = getProviderConfig loginHint >>= \case
   (Right issuerLoc, clientId) -> do
     unless ("https:" `T.isPrefixOf` issuerLoc
             || "http://localhost" `T.isPrefixOf` issuerLoc) $
-      throwIO $ TLSNotUsedException
-        $ "The issuer location doesn't start with 'https:'. \
-          \OIDC requires all communication with the IdP to use TLS."
+      throwIO $ TLSNotUsedException $ unwords
+        [ "The issuer location doesn't start with 'https:'. "
+        , "OIDC requires all communication with the IdP to use TLS."
+        ]
```

Comment:
This wasn't a style change - `CPP` doesn't play nicely with multi-line string literals, so I was forced to change them.